NaviGator/mission_control/navigator_missions/vrx_missions/mission_mux.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import txros
+from vrx import Vrx
+from scan_the_code import ScanTheCode
+
+class MissionMux(Vrx):
+    def __init__(self):
+        super(MissionMux, self).__init__()
+
+    @txros.util.cancellableInlineCallbacks
+    def run(self, args):
+        msg = yield self.task_info_sub.get_next_message()
+        task_name = msg.name
+        if task_name == 'stationkeeping':
+            yield self.run_station_keeping()
+        elif task_name == 'wayfinding':
+            yield self.run_wayfinding()
+        elif task_name == 'scan_and_dock':
+            scan_the_code = ScanTheCode()
+            yield scan_the_code.run(None)
```

Comment:
Makes more sense to just keep the mission called `Vrx`. You really just needed to add these two lines to vrx.py
```

            scan_the_code = ScanTheCode()
            yield scan_the_code.run(None)
```